### PR TITLE
refactor: add new fields to snapshot

### PR DESCRIPTION
**Pull request by witgaw**

### CODE REVIEW

File: core/liquidity/snapshot.go [hunk 1]
```diff
@@ -320,6 +347,29 @@ func (e *SnapshotEngine) serialiseSupplied() ([]byte, bool, error) {
...
+	scores := make([]*snapshotpb.LiquidityScore, 0, len(e.avgScores))
+	for k, v := range e.avgScores {
+		s := &snapshotpb.LiquidityScore{
+			PartyId: k,
+			Score:   v.String(),
+		}
+		scores = append(scores, s)
+	}
+
+	payload := &snapshotpb.Payload{
```
Comment (wwestgarth) [inline core/liquidity/snapshot.go:360]: Can you sort the `scores` slice by `PartyID` before putting it in th payload? It just makes sure the snapshot is deterministic.

Comment (ze97286) [inline core/liquidity/snapshot.go:360]: you must sort scores first.

Comment (ze97286) [reply core/liquidity/snapshot.go:360]: you made it sound like it was optional lol, I was more brutal :)

Comment (witgaw) [reply core/liquidity/snapshot.go:360]: Yep, done now.

**Pull request merged**